new_app/instructorDashboard.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
Also, thoughts on using jsx layouts + views? I was thinking something like this - https://github.com/reactjs/express-react-views